### PR TITLE
Feature1 : Display Case Types

### PR DESCRIPTION
**Pull request by selva08sankari**

### CODE REVIEW

File: src/main/java/in/casekartin/model/CaseManagerModel.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package in.casekartin.model;
+
+//create a service class
+public class CaseManagerModel {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/casekartin/model/CaseManagerModel.java:4]: Remove "Model" in suffix.

File: src/main/java/in/casekartin/model/CaseManagerModel.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	}
+
+	public Float getCost() {
+		return cost;
+	}
+
+	private String caseType;
+	private Float cost;
+	
+	private CaseManagerModel()
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/casekartin/model/CaseManagerModel.java:16]: Remove private constructor.

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -1,20 +1,13 @@
-<%@page import="java.util.Date"%>
-<%@page import="java.time.LocalDate"%>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt"%>
+
 <html>
 <head>
 <title>MyApp</title>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp:4]: Change title

**Review by ghost — CHANGES_REQUESTED** [review]
Added review comments

File: src/test/java/app/CaseManagerTest.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package app;
```
Comment (csys-fresher-batch-2021) [inline src/test/java/app/CaseManagerTest.java:1]: Follow package name.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**